packages/compass-web/scripts/sync-dist-to-mms.mjs

```diff
@@ -0,0 +1,166 @@
+import process from 'node:process';
+import fs from 'node:fs';
+import path from 'node:path';
+import child_process from 'node:child_process';
+import os from 'node:os';
+import util from 'node:util';
+import net from 'node:net';
+import timers from 'node:timers/promises';
+
+if (!process.env.MMS_HOME) {
+  throw new Error(
+    'Missing required environment variable $MMS_HOME. Make sure you finished the "Cloud Developer Setup" process'
+  );
+}
+
+function isDevServerRunning(port, host = '127.0.0.1') {
+  return new Promise((resolve) => {
+    const socket = new net.Socket();
+    socket.setTimeout(1000);
+    socket.once('connect', () => {
+      socket.destroy();
+      resolve(true);
+    });
+    socket.once('timeout', () => {
+      socket.destroy();
+      resolve(false);
+    });
+    socket.once('error', () => {
+      resolve(false);
+    });
+    socket.connect(port, host);
+  });
+}
+
+let devServer;
+if (!(await isDevServerRunning(8081))) {
+  console.log('mms dev server is not running... launching!');
+  child_process.execFileSync('pnpm', ['install'], {
+    cwd: process.env.MMS_HOME,
+    stdio: 'inherit',
+  });
```

Comment:
Can we either make this skippable or make it ignore strict engine requirement in mms? I usually keep nodejs version on the same version as electron for compass and it rarely matches what mms has